Makefile

```diff
@@ -36,6 +36,44 @@ publish-java11-local: build-java11
 		-v "${HOME}/.aws:/home/newrelic-lambda-layers/.aws" \
 		newrelic-lambda-layers-java11
 
+build-java17:
+	docker build \
+		--no-cache \
+		-t newrelic-lambda-layers-java17 \
+		-f ./dockerfiles/Dockerfile.java17 \
+		.
+
+publish-java17-ci: build-java17
+	docker run \
+		-e AWS_ACCESS_KEY_ID \
+		-e AWS_SECRET_ACCESS_KEY \
+		newrelic-lambda-layers-java17
+
+publish-java17-local: build-java17
+	docker run \
+		-e AWS_PROFILE \
+		-v "${HOME}/.aws:/home/newrelic-lambda-layers/.aws" \
+		newrelic-lambda-layers-java17
+
+build-nodejs14x:
```

Comment:
This PR contains Node 14 build/release jobs. These have been removed from this repository and should not be restored.

Reply:
Removed old node14 builds.